src/main/java/frc/robot/Constants.java

```diff
@@ -6,11 +6,38 @@
 /*----------------------------------------------------------------------------*/
 package frc.robot;
 
+
 public final class CONSTANTS {
 
     //General Constants
     public static final int PCM_ID = 3;
 
+   
+    public static final int TIMEOUT_MS = 30;
+    /**
+	 * Motor neutral dead-band, set to the minimum 0.1%.
+	 */
+    public static final double NEUTRAL_DEADBAND = 0.001;
+    public static final int kUnitsPerRevolution = 2048; /* this is constant for Talon FX */
+    
+    //Gains
+    public static final int TALON_PRIMARY_CLOSED_LOOP = 0;
+
+    //Turret Constants
+    public static final int turretEncoderLeftSoftLimit = 85; //degree of freedom, if not met seeking will be required
+    public static final int turretEncoderRightSoftLimit = -85; //degree of freedom, if not met seeking will be required
+
+    public static final int TURRET_PORT = 0;
```

Comment:
This should probably be named consistently with the naming in CTRE's documentation, calling this is a "slot" (even if their examples don't use it, confusingly). Perhaps TURRET_PID_SLOT as a name?